parse_test.go

```diff
@@ -72,6 +72,64 @@ func TestParseUntilEOF(t *testing.T) {
 	}
 }
 
+// TestParse_CEchoRQ is a test for parsing a CEchoRQ command. It checks that
+// the command is parsed correctly and that the expected tags are present in
+// the dataset.
```

Comment:
Can we include a motivation for adding this test?

```suggestion
// TestParse_CEchoRQ is a test for parsing a CEchoRQ command. It checks that
// the command is parsed correctly and that the expected tags are present in
// the dataset. These commands are typically very small dicom fragments, and 
// ensure that our parsing and transfer syntax inference logic work correctly.
```